.vib/elasticsearch/goss/elasticsearch.yaml

```diff
@@ -0,0 +1,11 @@
+command:
+  # ES7 does not support using `elasticsearch --version` without properly configuring it
+  check-app-version:
+    exec: ls /opt/bitnami/elasticsearch/lib/elasticsearch-core-*.jar | grep {{ .Env.APP_VERSION }}
+    exit-status: 0
+file:
```

Comment:
I would additionally check the `log4j2.properties` file, as it is affected by this piece of logic in the postunpack:

https://github.com/bitnami/containers/blob/main/bitnami/elasticsearch/8/debian-11/rootfs/opt/bitnami/scripts/libelasticsearch.sh#L913

Reply:
Added now, thanks!